metatron-app/metatron-discovery
# #3613 added saving grid width

### Description

It is inconvenient because it cannot be saved after changing the grid width.

**Related Issue** : #3613 


### How Has This Been Tested?

1. Go to the chart creation screen.
2. Draw a grid chart.
3. Change the width and save.
4. Check that the changed width is preserved.

#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
